devel/cctools/files/cctools-829-lto.patch

```diff
@@ -14,7 +14,7 @@
 -				    "xctoolchain/usr/lib/libLTO.dylib",
 -				    RTLD_NOW);
 +		const char *lto_paths[] = {
-+		    "@@LLVM_LIBDIR@@/libLTO.dylib",
++		    "/opt/local/libexec/llvm-11/lib/libLTO.dylib",
```

Comment:
Assuming you didn’t mean to hardcore the llvm version here

Reply:
oh, good pickup

Reply:
Good catch! No that's not intentional... guessing it's patched by a `reinplace` & I somehow missed that! I won't be able to fix this until Friday as I'm traveling without a work computer. If someone else wants to fix it go do it!